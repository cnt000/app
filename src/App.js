import React from 'react';
import { Counter } from './features/counter/Counter';
import Header from './layout/Header/Header';
import Menu from './layout/Menu/Menu';
import Plp from './layout/Plp/Plp';
import Pdp from './layout/Pdp/Pdp';
import Article from './layout/Article/Article';
import Footer from './layout/Footer/Footer';

import './App.scss';

const labels = {
  openMenuLabel: 'apri il menu',
  title: 'Pungilandia',
  titleDescription: 'Piante grasse, strane ed insolite',
  goToCartLabel: 'vai al carrello',
};

const App = () => (
  <div className="App">
    <Header {...labels} />
    <Menu />
    <Plp />
    <Pdp />
    <Article />
    <Footer />
    <Counter />
  </div>
);

export default App;
