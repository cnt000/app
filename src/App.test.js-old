import React from 'react';
import { render } from '@testing-library/react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import store from './store';
import App from './App';

test('renders `Test App` name', () => {
  const { getByText } = render(
    <Provider store={store}>
      <App />
    </Provider>,
  );
  expect(getByText(/Test App/i)).toBeInTheDocument();
});

it('renders correctly', () => {
  const tree = renderer
    .create(
      <Provider store={store}>
        <App />
      </Provider>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
