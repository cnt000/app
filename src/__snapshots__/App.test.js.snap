// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="App"
>
  <header
    className="header"
  >
    <div>
      <button
        className="burger"
      >
        apri il menu
      </button>
    </div>
    <div
      className="content"
    >
      <div
        className="logo"
      >
        <img
          alt="pungilandia logo"
          src="logo.png"
        />
      </div>
      <div
        className="title"
      >
        <h1>
          Pungilandia
        </h1>
        <p>
          Piante grasse, strane ed insolite
        </p>
      </div>
    </div>
    <div>
      <button
        className="cart"
      >
        vai al carrello
      </button>
    </div>
  </header>
  <div>
    Pungilandia Test App
  </div>
  <header
    className="header"
  >
    <div>
      <button
        className="burger"
      />
    </div>
    <div
      className="content"
    >
      <div
        className="logo"
      >
        <img
          alt="pungilandia logo"
          src="logo.png"
        />
      </div>
      <div
        className="title"
      >
        <h1 />
        <p />
      </div>
    </div>
    <div>
      <button
        className="cart"
      />
    </div>
  </header>
  <header>
    <h1>
      Menu
    </h1>
  </header>
  <div>
    <div
      className="row"
    >
      <button
        aria-label="Increment value"
        className="button"
        onClick={[Function]}
      >
        +
      </button>
      <span
        className="value"
      >
        0
      </span>
      <button
        aria-label="Decrement value"
        className="button"
        onClick={[Function]}
      >
        -
      </button>
    </div>
    <div
      className="row"
    >
      <input
        className="textbox"
        onChange={[Function]}
        value={2}
      />
      <button
        className="button"
        onClick={[Function]}
      >
        Add Amount
      </button>
    </div>
  </div>
  <div>
    domain: 
    http://localhost:8080
    <section>
      <h1>
        name: 
      </h1>
      <img />
      <br />
      <div>
        size: 
      </div>
      <div>
        price: 
      </div>
      <br />
      <a>
        Add
      </a>
      <br />
    </section>
  </div>
  <br />
  <div />
  <footer
    class="footer"
  >
    <p>
      <span
        class="copyright"
      >
        Copyright Â© 2020
      </span>
       -
      <a
        class="privacy"
        href="http://www.pungilandia.com"
      >
        Privacy
      </a>
       
      -
      <a
        class="contactus"
        href="mailto:info@pungilandia.com"
      >
        Contatti
      </a>
    </p>
    <address>
      Azienda Agricola Pungilandia Rocchi Domenico
      <br />
      Via Isonzo 11, 65123 Pescara P.IVA 01203140684 CF RCCDNC59R30I712B Iscrizione REA Pescara Nr.97422
    </address>
    <p
      class="laws"
    >
      Azienda in possesso Registro di Detenzione di Specie Vegetali Incluse Negli Allegati CITES A e B del Regolamento (CE) 338/97 del Consiglio del 9 Dicembre 1996 e Successive Modifiche e Attuazioni. Registro N.000001/2001 VAB
    </p>
  </footer>
</div>
`;
